ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -20,15 +20,15 @@
 - hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
-    - name: Create and provision external computes
-      ansible.builtin.include_role:
-        name: libvirt_manager
-        tasks_from: deploy_edpm_compute.yml
-
-    - name: Deploy EDPM
-      ansible.builtin.include_role:
-        name: edpm_deploy
-      when: deploy_edpm | default('false') | bool
+    - name: Install requirements
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
+        target: edpm_compute_baremetal
```

Comment:
Please change this task to something like this.
```
  ansible.builtin.include_role:
    name: 'install_yamls_makes'
    tasks_from: 'edpm_compute_baremetal'
```
Because above task will not work, under ci_framework makefile there is no edpm_compute_baremetal target.